extensions/interactions/AlgebraicExpressionInput/AlgebraicExpressionInput.py

```diff
@@ -53,3 +54,242 @@ class AlgebraicExpressionInput(base.BaseInteraction):
         },
         'default_value': False
     }]
+
+    @classmethod
+    def to_proto(
+        cls, default_outcome, customization_args, solution, hints, answer_groups
+    ):
+        """Creates a AlgebraicExpressionInputInstanceDto proto object.
+
+        Args:
+            default_outcome: Outcome. The domain object.
+            customization_args: CustominzationArgs. The domain object.
+            hints: Hint. The domain object.
+            solution: Solution. The domain object.
+            answer_groups: AnswerGroups. The domain object.
+
+        Returns:
+            AlgebraicExpressionInputInstanceDto. The proto object.
+        """
+        customization_args_proto = (
+            cls._to_customization_args_proto(customization_args)
+        )
+        outcome_proto = default_outcome.to_proto()
+        hints_proto_list = cls.get_hint_proto(cls, hints)
+        solution_proto = cls._to_solution_proto(solution)
```

Comment:
Typically, "to_dict", "to_proto" etc. are instance methods -- e.g. object.to_proto().

For something like this I think you'll want something like cls._convert_solution_to_proto() instead.

Reply:
Fixed